lib/resque-retry/server.rb

```diff
@@ -66,15 +66,23 @@ def local_template(path)
       # cancels job retry
       def cancel_retry(job)
         klass = get_class(job)
-        retry_key = retry_key_for_job(job)
-        Resque.remove_delayed(klass, *job['args'])
-        Resque.redis.del("failure-#{retry_key}")
-        Resque.redis.del(retry_key)
+        if klass
+          retry_key = retry_key_for_job(job)
+          Resque.remove_delayed(klass, *job['args'])
+          Resque.redis.del("failure-#{retry_key}")
+          Resque.redis.del(retry_key)
+        else
```

Comment:
Would we really want to re-raise an untyped exception here? If we could do this a bit more gracefully that'd be preferable but I think this would require a little more work on the Web UI side.